backend/loi-service/src/main/java/org/egov/works/config/LOIConfiguration.java

```diff
@@ -0,0 +1,74 @@
+package org.egov.works.config;
+
+import lombok.*;
+import org.egov.tracer.config.TracerConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Import;
+import org.springframework.stereotype.Component;
+
+@Getter
+@Setter
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Import({TracerConfiguration.class})
+@Component
+public class LOIConfiguration {
+    //Workflow Config
+    @Value("${egov.workflow.host}")
```

Comment:
Workflow host config has been repeated below. Remove it from here. 
Check if there are any other repeatitions.

Reply:
Done